src/implementations/config/types.ts

```diff
@@ -1,10 +1,15 @@
 import { v2 } from "@govtechsg/open-attestation";
+import { Wallet } from "ethers";
 
-type WalletEncrypted = {
-  type: string;
+type WalletEncryptedJson = {
+  type: "ENCRYPTED_JSON";
   encryptedJson: string;
 };
 
+type WalletOptions = Wallet | WalletEncryptedJson;
```

Comment:
we dont use wallet type anymore right? are u catering for older versions? im just thinking we should stick to the newer version, walletEncryptedJson

Reply:
ok